MdeModulePkg/Library/BaseIpmiCommandLibNull/IpmiCommandLibNetFnSensor.c

```diff
@@ -0,0 +1,48 @@
+/** @file
+  IPMI Command - NetFnApp NULL instance library.
+
+  Copyright (c) 2018 - 2021, Intel Corporation. All rights reserved.<BR>
+  Copyright (C) 2023 Advanced Micro Devices, Inc. All rights reserved.<BR>
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+#include <Uefi.h>
+#include <IndustryStandard/Ipmi.h>
+
+/**
+ * Send the Set Sensor Threshold command for a specified SensorNumber
+ *
+ * @param[in]   SetSensorThresholdRequestData   The filled out Set Sensor Threshold command structure
+ * @param[out]  CompletionCode                  Pointer to a buffer for returning the completion code
+ *
+ * @retval  EFI_SUCCESS
+ * @retval  EFI_INVALID_PARAMETER
+ */
+EFI_STATUS
+EFIAPI
+IpmiSetSensorThreshold (
+  IN IPMI_SENSOR_SET_SENSOR_THRESHOLD_REQUEST_DATA  *SetSensorThresholdRequestData,
+  OUT UINT8                                         *CompletionCode
+  )
+{
+  return RETURN_UNSUPPORTED;

```

Comment:
The `EFI_UNSUPPORTED` return value should be accounted for in the function description.

Reply:
Taken care of this return value EFI_UNSUPPORTED